code/style.css

```diff
@@ -1,4 +1,244 @@
-html {
-  background: #0025ff;
-  color: #fff;
+html, body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Noticia Text', serif;
+  overflow-x: hidden;
 }
+
+
+/*Mobile first design approach*/
+
+ @media only screen and (min-width: 320px) and (max-width: 767px){
+  .big-news-text {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    margin-top: 5px;
+    color: white;
+    font-size: 20px;
+    background-color: black;
+  }
+
+  .big-news article {
+    position: relative;
+  }
+
+  nav a{
+    display: none;
+  }
+ }
+/* Mobile */
+@media only screen and (min-width: 320px) {
```

Comment:
This seems to apply to everything, i.e. mobile, tablet and desktop so why is the media query needed? It looks like you have the mobile styling specified starting on line 11, see below:
```css
 @media only screen and (min-width: 320px) and (max-width: 767px)

Reply:
True! Thanks a lot for pointing it out, I figured how to do it for w.4 project :)